helm/gcp/values.yaml

```diff
@@ -452,34 +452,82 @@ authOptions:
     server: "ldap://pl-dev-glauth.dev-gke.svc.cluster.local:3893"
     dn: "cn=%u,ou=users,ou=users,dc=pldev,dc=io"
     startTls: false
+    # -- LDAP search password configuration.
+    # Used for LDAP bind operations when searching for users.
+    # The password can be provided via a Kubernetes Secret as an environment variable (PL_AUTH_LDAP_SEARCH_PASSWORD).
+    # Alternatively, you can pass it directly via the --auth-ldap-search-password CLI argument in extraArgs.
+    searchPassword:
+      # -- Provide search password from a Secret as an environment variable.
+      # When enabled, the password will be available as PL_AUTH_LDAP_SEARCH_PASSWORD env var.
+      # Note: If you prefer to pass the password directly via CLI, use extraArgs instead:
+      # extraArgs:
+      #   - "--auth-ldap-search-password=your-password"
+      envRef:
+        enabled: false
+        name: "" # Name of the Secret containing the LDAP search password
+        key: "password" # Key within the Secret that holds the password value
     # -- LDAP TLS configuration.
     # Enables client-side TLS for the LDAP connection.
     # You can mix and match how certificates are provided. For example, you can
     # provide the client cert/key via a secret and the CA certificate via `casPath`.
     tls:
       enabled: false
-      # -- Provide certificate, key, and optional CA via direct paths on the container's filesystem.
-      # This is used if `tls.secretRef.enabled` is false.
-      certPath: ""
-      keyPath: ""
-      casPath: "" # Optional. Path to a CA certificate file.
-      # -- Provide certificate, key, and optional CA via a Kubernetes secret.
-      # The secret will be mounted to `/etc/platforma/secrets/ldap`.
+      # -- Configuration for the Certificate Authority (CA).
+      # Used to verify the LDAP server's certificate. This is useful for servers
+      # with self-signed certificates or private CAs.
+      ca:
```

Comment:
Just a note: will remove this in next version of backend. The only thing that will remain is what is configured here as 'root-cas' option.